articles.sql

```diff
@@ -0,0 +1,11 @@
+\c nc_news_test
+
```

Comment:
This scrap code shouldnt really be part of the final commit. 
I'd make sure you're adding the files you want to submit, or using .gitignore to ignore scrap code.